manifests/manifest-base.yml

```diff
@@ -2,8 +2,10 @@
 buildpack: go_buildpack
 memory: 256M
 services:
-  - deck-ups
+- deck-ups
+- deck-redis
```

Comment:
We're going to rename `deck-redis` to `dashboard-redis` but we don't wan to touch `deck-ups`. I'm around to pair on that change if you'd like though @msecret.